api/msi.h

```diff
@@ -106,11 +106,11 @@ int msi_epetra_print(int* matrix_id);
 
 int msi_solver_aztec(int* matrix_id, pField x_field, pField b_field,
                        int* num_iter, double* tolerance,
-		       const char* krylov_solver, const char*
-		       preconditioner, const char* sub_dom_solver,
-		       int* overlap, int* graph_fill, double*
-		       ilu_drop_tol,  double* ilu_fill,
-		       double* ilu_omega, int* poly_ord);
+           const char* krylov_solver, const char*
```

Comment:
This file appears to only have whitespace changes. If formatting needs to be fixed (tabs to two spaces, etc.) then please submit a separate PR with commits that specifically address it.